src/process_wrap.cc

```diff
@@ -163,9 +169,11 @@ class ProcessWrap : public HandleWrap {
       int argc = js_argv->Length();
       // Heap allocate to detect errors. +1 is for nullptr.
       options.args = new char*[argc + 1];
+      CHECK_NE(options.args, nullptr);
```

Comment:
`new ...` won't return nullptr on failure, it throws a `std::bad_alloc` exception (that's effectively uncatchable because we build with `-fno-exceptions`.)